### ISSUE TITLE
Request for Vietnamese Bible 2010 version

### ISSUE THREAD

**Issue by minhtrung1997** [godlytalias/Bible-Database#31]
I believe that having this resource easily accessible would greatly benefit the Vietnamese Christian community and those studying the Bible in Vietnamese.

I understand that there may be copyright or licensing considerations involved. If possible, I would be happy to discuss potential options for making the Bible available in a way that respects these concerns.

Thank you for your time and consideration.

**Comment by godlytalias** [comment]
Hello,Sure, I will get back on this. Can you please raise an issue in the github repo so that I wont miss this out?
Thanks,Godly

Yahoo Mail: Search, organise, conquer 
 
  On Mon, 30 Sept 2024 at 10:09 am, ***@***.***> wrote:   


I believe that having this resource easily accessible would greatly benefit the Vietnamese Christian community and those studying the Bible in Vietnamese.

I understand that there may be copyright or licensing considerations involved. If possible, I would be happy to discuss potential options for making the Bible available in a way that respects these concerns.

Thank you for your time and consideration.

**Comment by minhtrung1997** [comment]
Yeah, issue has been raised in this repo. So what I need to do more?
